ulog/sentry/sentry.go

```diff
@@ -44,6 +44,18 @@ var _zapToRavenMap = map[zap.Level]raven.Severity{
 	zap.FatalLevel: raven.FATAL,
 }
 
+// Configuration provides sentry DSN and other optional parameters for logging
+type Configuration struct {
+	DSN      string
+	MinLevel *string                `yaml:"min_level,omitempty"`
```

Comment:
Right, I've read the docs and assumed both ways, but tested and it is unnecessary for parsing.